spec/integration/rom/factory_spec.rb

```diff
@@ -647,7 +660,7 @@
       end
 
       it 'allows overrides' do
-        user = factories[:user, name: "Joe"]
+        user = factories[:user, first_name: "Joe"]
```

Comment:
This is the exact kind of typo I would like to see an error for. I updated `name` to be `first_name` but we can just as easily make this be `user = factories[:user]`. I was not sure what the intent was for updating the `name` attribute considering there is no specific test that checks the update.